Parsers/ASimFileEvent/test/Microsoft_Security_Event_ASimFileEvent_SchemaTest.csv

```diff
@@ -0,0 +1,106 @@
+﻿Result
```

Comment:
(0) Error: 4 invalid value(s) (up to 10 listed) in 651155 records (99.24%) for field [EventType] of type [Enumerated]: ["ObjectDeleted","MetadataModified","ObjectModified","MetadataAccessed"] (Schema:FileEvent)

Reply:
added supported in tester file